bin/omarchy-install-password-manager

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if (($# == 0)); then
+  echo "Usage: omarchy-install-password-manager [1password|bitwarden|keepassxc] [extra_packages]"
+  exit 1
+fi
+
+package="$1"
+package_extras="$2"
+
+# Install package
+omarchy-pkg-add $package $package_extras
+
+if [ $package = "bitwarden" ]; then
```

Comment:
since we already need to check things for individual managers, I think the extra_packages param could be removed and we just check in this file if it's bitwarden and add the cli to the install command. that will cleanup the usage in the menu